ci: spurious output / errors in IWYU job

https://github.com/bitcoin/bitcoin/actions/runs/26292243180/job/77395542650:
Looks like IWYU is trying to run on libmultiprocess related code, and failing, but we shouldn't be trying to lint a subtree in the first place:
```bash
 /home/admin/actions-runner/_work/_temp/src/ipc/libmultiprocess/test/mp/test/test.cpp:5:10: fatal error: 'mp/test/foo.capnp.h' file not found
    5 | #include <mp/test/foo.capnp.h>
      |          ^~~~~~~~~~~~~~~~~~~~~

```
Along with other sporadic error output:
```bash
[1](https://github.com/bitcoin/bitcoin/actions/runs/26292243180/job/77395542650#step:11:11661)
error: no such file or directory: '/home/admin/actions-runner/_work/_temp/build/src/ipc/capnp/common.capnp.c++'
error: no input files
error: unable to handle compilation, expected exactly one compiler job in ''
error: no such file or directory: '/home/admin/actions-runner/_work/_temp/build/src/ipc/capnp/common.capnp.proxy-client.c++'
error: no input files
error: unable to handle compilation, expected exactly one compiler job in ''
error: no such file or directory: '/home/admin/actions-runner/_work/_temp/build/src/ipc/capnp/common.capnp.proxy-server.c++'
error: no input files
error: unable to handle compilation, expected exactly one compiler job in ''
error: no such file or directory: '/home/admin/actions-runner/_work/_temp/build/src/ipc/capnp/common.capnp.proxy-types.c++'
error: no input files
```

Ideally we'd have no sporadic errors / output, and wouldn't be trying to run linters/tools over subtrees at all.

I think that any errors in CI output/tooling should also be getting turned into real errors, so that they aren't missed/ignored?

--- a/cmake/libmultiprocess.cmake
+++ b/cmake/libmultiprocess.cmake
@@ -34,5 +34,5 @@ function(add_libmultiprocess subdir)
   # exclusion, tools like clang-tidy and IWYU that make use of compilation
   # database would complain that the generated c++ source files do not exist. An
   # alternate fix could build "mpexamples" by default like "mptests" above.
-  set_target_properties(mpcalculator mpprinter mpexample PROPERTIES EXPORT_COMPILE_COMMANDS OFF)
+  set_target_properties(mpcalculator mpprinter mpexample mptest PROPERTIES EXPORT_COMPILE_COMMANDS OFF)
 endfunction()

